operational_analysis/methods/plant_analysis.py

```diff
@@ -714,37 +856,56 @@ def sample_long_term_reanalysis(self, n, r):
             1. The reanalysis product
             2. The number of years to use in the long-term correction
 
-        A memoized approach is used here since there are a finite combination of long-term wind speeds:
-            (3 reanalysis products x 10 different year combinations) = 30 total combinations
-
-        This memoized approach saves significant computaitonal time in the Monte Carlo simulation
-
         Args:
-           n(:obj:`integer`): The Monte Carlo Iteration Number
-           r(:obj:`string`): The reanalysis product used for Monte Carlo sample 'n'
+           n(:obj:`integer`): The number of years for the windiness correction
+           r(:obj:`string`): The reanalysis product used for Monte Carlo sample False
```

Comment:
I think the "False" was a result of replace all 'n' to 'False'. It should be removed here.